### DISCUSSION TITLE
dropped 'float: left'

### DISCUSSION THREAD

**Discussion by hgfrey** [primefaces/community#1486] [PrimeFaces] [answered]
Was the 'float: left' intentionally dropped in version 14.0.0-RC2? Here you can see 13.0.7:
![image](https://github.com/primefaces/community/assets/33621098/c450569d-9b88-4ad4-ad3a-681f917e2e3f)

And here is 14.0.0-RC2:
![image](https://github.com/primefaces/community/assets/33621098/58467ceb-80c3-4e50-b724-85e8db699865)

**Comment by NicolaIsotta** [comment, marked as answer]
Commit is https://github.com/primefaces/primefaces/commit/61219c0d5c77cf0105749a58801b3f16ded46a15 by @melloware

**Reply by melloware** [reply]
Thanks @NicolaIsotta yep done in purpose for https://github.com/primefaces/primefaces/issues/10696 i don't see why `float:left` should ever be in there as floating is not responsive.

**Reply by hgfrey** [reply]
Do you have a hint for me as to what I can use instead?

**Reply by melloware** [reply]
you can simply add the style back in with your own CSS.

```css
.ui-g-1 {
  float:left;
}
```

Etc
